core/src/ast_builder/expr/function.rs

```diff
@@ -440,6 +447,9 @@ pub fn ifnull<'a, T: Into<ExprNode<'a>>, U: Into<ExprNode<'a>>>(expr: T, then: U
 pub fn ceil<'a, T: Into<ExprNode<'a>>>(expr: T) -> ExprNode<'a> {
     ExprNode::Function(Box::new(FunctionNode::Ceil(expr.into())))
 }
+pub fn rand<'a, T: Into<ExprNode<'a>>>(expr: T) -> ExprNode<'a> {
+    ExprNode::Function(Box::new(FunctionNode::Rand(Some(expr.into()))))
+}
```

Comment:
Currently, using ast builder we cannot use random function without seed.
It would be better to take `Option<ExprNode<'a>>` rather than `Into<ExprNode<'a>>`.
Like.. `rtrim` and `ltrim`.

fyi.
For now, when we use `Option<ExprNode<'a>>`, then `Into` is not supported yet.